doc/source/io.rst

```diff
@@ -2852,23 +2852,53 @@ Parsing Specific Columns
 
 It is often the case that users will insert columns to do temporary computations
 in Excel and you may not want to read in those columns. ``read_excel`` takes
-a ``usecols`` keyword to allow you to specify a subset of columns to parse.
+either a ``usecols`` or ``usecols_excel`` keyword to allow you to specify a
+subset of columns to parse. Note that you can not use both ``usecols`` and
+``usecols_excel`` named arguments at the same time.
+
+If ``usecols_excel`` is supplied, then it is assumed that indicates a comma
+separated list of Excel column letters and column ranges to be parsed.
+
+.. code-block:: python
+
+   read_excel('path_to_file.xls', 'Sheet1', usecols_excel='A:E')
+   read_excel('path_to_file.xls', 'Sheet1', usecols_excel='A,C,E:F')
```

Comment:
For consistency I think we should *only* allow A1 notation, which the second example here does not do. Instead I think this should be written as

```python
read_excel('path_to_file.xls', 'Sheet1', usecols_excel='A:A,C:C,E:F')
```

Raises for anything else.  See

https://docs.microsoft.com/en-us/previous-versions/office/developer/office-2007/bb211395(v=office.12)

Reply:
I think that despite of the second example does not follow exactly the A1 notation it's simpler to write it.

@jreback, what do you think?

Reply:
@jreback and @chris-b1, can you review this, this https://github.com/pandas-dev/pandas/pull/20480#discussion_r194238427 and this https://github.com/pandas-dev/pandas/pull/20480#issuecomment-395975007, please?

Reply:
@jreback and @chris-b1, can you review this https://github.com/pandas-dev/pandas/pull/20480#discussion_r194238455, this https://github.com/pandas-dev/pandas/pull/20480#discussion_r194238427 and this https://github.com/pandas-dev/pandas/pull/20480#issuecomment-395975007, please?

Reply:
We already support the single column `'A'` vs `'A:A'` so fine with this